src/commands/talk.test.ts

```diff
@@ -40,13 +40,13 @@ describe('Talk Slash Command', () => {
 			reply: mockReply,
 		} as unknown as TextInputCommandContext;
 		mockSay.mockReturnValue(emptyBuffer);
+		mockGetString.mockReturnValue(message);
+		mockGetInteger.mockReturnValue(null);
 	});
 
 	test('Fails if no options are provided', async () => {
-		context = {
-			...context,
-			interaction: { options: { data: { length: 0 } } },
-		} as unknown as TextInputCommandContext;
+		mockGetInteger.mockReturnValueOnce(null);
+		mockGetString.mockReturnValueOnce(null);
```

Comment:
Given that in `talk.ts`, there is an explicit test for `if (options.data.length === 0)`, then wouldn't replacing this prevent that line from getting test coverage?

Reply:
Lemme check...

Reply:
I agree that this new system is better though, so if anything, that `if` statement should be removed/changed.